study-builder/studies/rarex/add-participant.conf

```diff
@@ -320,6 +320,22 @@
               "templateCode": null,
               "templateText": "",
               "variables": []
+            },
+            "picklistLabelTemplate": {
```

Comment:
I think adding the changes directly to the config files is nice, but one thing you lose out on is the ability to test out the different versions of the activities locally, since the config files now reflect the latest thing and there's no config files for the first version. Once we refactor the `RarexVersion2` task to do revisioning properly, I recommend not changing the config files.

Since the `RarexVersion2` should be doing the revisioning correctly, what we can do is then add `RarexVersion2` to the study's `patch-log.conf`. So then, in local development, we can get the first version of things, and then run `--patch` to apply `RareXVersion2` and get version 2 of things. For deployed environments we can `--run-task RarexVersion2` manually to get deployed study up-to-date.

Reply:
Yes, you're right, however my thoughts were: we can checkout any version of configuration for deployment if needed. But once all the environments have updated with patch we don't actually need the "old" version. Also, making next patch will be easier looking at configuration files rather than having updated environment only.
What do you think?

Reply:
That sounds good. I think it will work better if studies are in their own repos so it's much easier to go off of git commits/tags to checkout an older version of things. But right now studies are all in the same repo. Maybe create a new git tag like `rarex-v1` or something so we actually know which commit to checkout?

Reply:
I think that would work